website/content/en/status/report-2025-04-2025-06/intro.adoc

```diff
@@ -0,0 +1,14 @@
+Here is the second 2025 status report, with 32 entries.
+
+As for the preceding quarters, this report is published just a few days before calls for 2025Q3 report submissions are sent.
+Indeed, although according to our link:https://docs.freebsd.org/en/articles/freebsd-status-report-process/#_timeline[timeline] we should have published this report in July (general rule is publication should happen within the month just after the calls for reports are sent), we kept receiving important reports until the end of August.
+This is both a positive and a negative thing: on a hand, it means that our FreeBSD community is busy fixing existing issues and implementing new features, making the OS we love better and better every day, that it works so intensely that only little time is left for submitting reports; however on the other hand it also means that news in these reports are always published when they are already two months old.
+Two months is not bad, especially if we consider that FreeBSD communication happens on many other channels too, but it would be nice if we could improve it.
+
+If you are a late submitter, please take some time to evaluate if there is anything you can do to improve your report submission punctuality.
+The status team is always glad to do anything that you can need to ease the submission process: if there is something we can do for you, just ask.
+If you are a contributor or just a FreeBSD user, please consider contributing more if you can: even working on one small simple task only is useful, it can help lower the pressure on other developers, for whom it might thus become easier to find the time to document their work for the community.
```

Comment:
```suggestion
If you are a contributor or just a FreeBSD user, please consider contributing more if you can.
Even working on a single small simple task is useful, it can help to lower the pressure on other developers, for whom it might thus become easier to find the time to document their work.
```

Reply:
Shorter sentences.